libraries/AP_BattMonitor/AP_BattMonitor_SMBus.h

```diff
@@ -23,8 +23,7 @@ class AP_BattMonitor_SMBus : public AP_BattMonitor_Backend
     // virtual destructor to reduce compiler warnings
     virtual ~AP_BattMonitor_SMBus() {}
 
-    // all smart batteries provide current and individual cell voltages
-    bool has_current() const override { return true; }
+    // all smart batteries are expected to provide current
     bool has_cell_voltages() const override { return true; }
```

Comment:
You removed the wrong one. Here is what you need: :sleeping_bed: :sleeping:
 :smile: